client/src/components/MyAvailabilityCards.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+// import DeleteMyAvailability from "./DeleteMyAvailability";
+// import EditMyAvailability from "./EditMyAvailability";
```

Comment:
If we are not using those imports - let's delete them.

Reply:
We are using them I have uncommented them. Thank you.